src/pattern-library/components/PlaygroundApp.js

```diff
@@ -50,23 +50,26 @@ export default function PlaygroundApp({
   }
 
   return (
-    <main className="PlaygroundApp">
-      <div className="PlaygroundApp__sidebar">
-        <div className="PlaygroundApp__sidebar-home">
+    <main className="grid grid-cols-[20rem_1fr] w-screen h-screen">
```

Comment:
If you're new to the syntax, square brackets allow the use of arbitrary values in certain Tailwind utility classes. This obviates the need to enshrine this grid layout in the configuration, as it is a one-off.